dts/bindings/modem/ublox,sara-r4.yaml

```diff
@@ -0,0 +1,37 @@
+#
```

Comment:
You'll need a commit message in commit header

Reply:
Ah yes, I'll fix.  Thanks.